libbpf-rs/tests/bin/src/cgroup-fw.bpf.c

```diff
@@ -0,0 +1,153 @@
+#include <linux/bpf.h>
```

Comment:
Thanks for adding this test object! It seems fine conceptually, but can we please boil it down to the essence? This seems more like an example than something we want in a basic unit test. E.g., I don't think we need all the filtering logic. It's probably easier to just use `__u32` values instead of 16 byte `pkt` ones.

Perhaps we don't need a new object altogether and can just add a queue/stack to, say, `tracepoint.bpf.c`? What do you think?

Reply:
my bad, I just copied the file from the project I'm working on. I added a queue map and a stack map to the tracepoint.bpf.c